JS/audio.js

```diff
@@ -0,0 +1,37 @@
+function listenKeypress() {
+    document.addEventListener('keypress', function(e) {
+        var keys = {
+            'q': '1',
+            'w': '2',
+            'e': '3',
+            'r': '4'
+        }
+        if (e.key in keys) {
+            var sound = 'cheer' + keys[e.key]
+            makeNoise(sound)
+            var $city = $('#city-' + keys[e.key])
+            shootLaser($city)
+        }
+    })
+}
+
+var cities = document.querySelectorAll('.city')
```

Comment:
why is the keypress event inside a function but this one isnt??

Reply:
removed in 20d4e79